packages/cli/src/ui/commands/bugCommand.ts

```diff
@@ -47,16 +129,50 @@ export const bugCommand: SlashCommand = {
 `;
 
     let bugReportUrl =
-      'https://github.com/google-gemini/gemini-cli/issues/new?template=bug_report.yml&title={title}&info={info}';
+      'https://github.com/google-gemini/gemini-cli/issues/new?template=bug_report.yml&title={title}&info={info}&problem={problem}';
 
     const bugCommandSettings = config?.getBugCommand();
     if (bugCommandSettings?.urlTemplate) {
       bugReportUrl = bugCommandSettings.urlTemplate;
     }
 
-    bugReportUrl = bugReportUrl
+    let url = bugReportUrl
       .replace('{title}', encodeURIComponent(bugDescription))
-      .replace('{info}', encodeURIComponent(info));
+      .replace('{info}', encodeURIComponent(info))
+      .replace('{problem}', encodeURIComponent(problem));
+
+    let wasTruncated = false;
+    while (url.length > 8000 && subsequentItems.length > 0) {
+      wasTruncated = true;
+      subsequentItems.shift();
+      const truncatedCliResponses = formatCliResponses(subsequentItems);
+      const truncatedLastCliResponse = truncatedCliResponses.length
+        ? truncatedCliResponses
+        : '... truncated response due to character limit ...';
+
+      const problemText =
+        truncatedCliResponses.length > 0 && wasTruncated
+          ? `... truncated response due to character limit ...\n${truncatedLastCliResponse}`
+          : truncatedLastCliResponse;
+
+      const truncatedProblem = [
+        '**Last User Prompt**',
+        '```',
+        lastUserPrompt?.text || 'N/A',
+        '```',
+        '',
+        '**Last Gemini CLI Response**',
+        '```',
+        problemText,
+        '```',
+      ].join('\n');
+      url = bugReportUrl
+        .replace('{title}', encodeURIComponent(bugDescription))
+        .replace('{info}', encodeURIComponent(info))
+        .replace('{problem}', encodeURIComponent(truncatedProblem));
+    }
+
+    bugReportUrl = url;
 
     context.ui.addItem(
```

Comment:
So one note: it's possible the `bugReportUrl` gets too big. All of the content gets embedded into the query string which may blow out the browsers query string limit. We'll want to add protection.

In addition for this addItem we'll also want to indicate to the user if we have included their request / response in the report and to instruct them to remove the content if needed.